CFC_DataCollector/moves/collect.py

```diff
@@ -87,6 +87,7 @@ def fillSectionWithMovesData(sec_from_moves, newSec):
    newSec['duration'] = sec_from_moves["duration"] if "duration" in sec_from_moves else None
    newSec['distance'] = sec_from_moves["distance"] if "distance" in sec_from_moves else None
    newSec['track_points'] = [{'track_location':{'type':'Point', 'coordinates':[point["lon"],point["lat"]]}, 'time':point["time"]}for point in sec_from_moves["trackPoints"]] if "trackPoints" in sec_from_moves else []
+   newSec['cleaned_points'] = _cleanGPSData(newSec['track_points'])
```

Comment:
This won't work, because the rest of the code (ML pipeline, client map display...) all use `track_points`. So this will include cleaned points, but won't affect anything else.

In order to really fix this, you need to:

1. change all the uses of `track_points` to use `cleaned_points` instead, OR
2. change this so that the *cleaned* points are stored in `track_points` and the original points are stored in `original_points` or something.

This will ensure that all the rest of the code works with the cleaned points, but we still have the original points around for further analysis.